site/app/templates/forum/CreatePost.twig

```diff
@@ -109,6 +109,16 @@
 
         <div id="{{ post_attachment.params.well_id }}" class="attachment-well well well-lg"></div>
 
+        {# Checks whether the page is on TA grading or the Discussion forum and opens attachments based off that#}
+        <script>
+            function openAttachments() {
+                if ( window.location.pathname == '/courses/s23/hashtag8471/gradeable/hw01/grading/grade') {
```

Comment:
This is hard-coded to only your course on your development environment. It would probably also be nicer to put this logic in the TA grading JavaScript code instead.